landing_page/templates/landing_page/base.html

```diff
@@ -1,23 +1,75 @@
-{% load static %}
-{% load static tailwind_tags %}
+{% load static %} {% load static tailwind_tags %}
 <!DOCTYPE html>
 <html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/c60455fa26.js"
+      crossorigin="anonymous"
+    ></script>
```

Comment:
i think if you add a newline b/w the two opposite angled brackets (>'\n'<\script>), the closing script tag should go on the next line and prettier should be okay with that.
just a personal preference thing to look out for in the future (if you care to)

Reply:
I'll keep that in mind. I think prettier did that itself but I'll change it.

Reply:
yeah i think it does, but think that if you force it otherwise, it should switch? maybe there's a setting for it - i can double check my work set-up tomorrow!